pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -121,6 +125,28 @@ func (np *Processor) updatePlacementInContract(nm netmap.NetMap, l *zap.Logger)
 
 		err = np.containerWrp.UpdateContainerPlacement(cID, vectors, replicas)
 		if err != nil {
+			expBackoff := backoff.NewExponentialBackOff()
+			expBackoff.InitialInterval = 1 * time.Second
+
+			err = backoff.RetryNotify(
+				func() error {
+					err = np.containerWrp.UpdateContainerPlacement(cID, vectors, replicas)
+					if errors.Is(err, client.ErrConnectionLost) {
+						return backoff.Permanent(err)
```

Comment:
from the outside it usually looks the other way around: the network channel is a temporary problem, and most of the others are permanent

client does not try to reconnect?